options/locale/locale_en-US.ini

```diff
@@ -3831,7 +3831,7 @@ runners.reset_registration_token_success = Runner registration token reset succe
 runs.all_workflows = All Workflows
 runs.commit = Commit
 runs.scheduled = Scheduled
-runs.pushed_by = pushed by
+runs.pushed_by = Pushed by
```

Comment:
IIRC it is used in a context, the old i18n keys are not well designed, but I guess we should keep the old text